### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by gavinhenderson** [AceCentre/FaceCommander#9]

**Comment by sjjhsjjh** [comment]
Hi, I had a quick click through the EyeCommander pages. Looks like they are mostly instructional videos. Is that also what's wanted for FaceCommander?

**Comment by gavinhenderson** [comment]
Yeah I have found videos to be most helpful for our target audience from past experience. I am happy to do all these videos, but it probably makes more sense to do it once we are ready to launch it so that the videos reflect the most up to date version.
